src/main/ressources/META-INF/persistence.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence version="2.1"
+	xmlns="http://xmlns.jcp.org/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
+	<persistence-unit name="JavaEE-JPA-resource-local-Servlets"
```

Comment:
À changer à terme pour utiliser services du conteneur (cf. exercices JPA plus avancés).

Reply:
Nous avions trouvé plus judicieux d'utiliser les services du conteneur dans le cadre de la fonctionnalité UseDB de l'itération 2.